dto/src/main/java/com/bankbazaar/dto/model/TripActivityDto.java

```diff
@@ -1,7 +1,9 @@
 package com.bankbazaar.dto.model;
+import com.bankbazaar.core.model.ActivityStatus;
```

Comment:
I dont think this is being used anywhere

Reply:
this has been removed